### PR TITLE
Charter 2023

### PR DESCRIPTION
**Pull request by chrisn**
The [current MEIG charter](https://www.w3.org/2021/06/me-ig-charter.html) expires on 30 April 2023. This PR is for a new charter for 2023-2025.

We welcome your feedback, comments, and suggestions on the group's priorities for the next charter period. Please feel free to raise issues in GitHub or comment in this PR.

[HTML preview](https://htmlpreview.github.io/?https://github.com/w3c/media-and-entertainment/blob/charter-2023/charters/charter-2023.html)

### CODE REVIEW

**Comment by chrisn** [comment]
We currently have [Media Integration Guidelines](https://w3c.github.io/me-media-integration-guidelines/) listed as a draft deliverable. Should we still plan to work on this? If not, we should probably remove it from the charter.

@JohnRiv @fghilardi

**Comment by JohnRiv** [comment]
> We currently have [Media Integration Guidelines](https://w3c.github.io/me-media-integration-guidelines/) listed as a draft deliverable. Should we still plan to work on this? If not, we should probably remove it from the charter.

Although it disappoints me to type this because we did a good amount of research for that note, I don't see that being a priority in the next 2 years that this renewed charter will be active, so I'd recommend we remove it. 

I'd still keep the [repo](https://github.com/w3c/me-media-integration-guidelines) around since there is valuable information in the [issues](https://github.com/w3c/me-media-integration-guidelines/issues) to work from if we were to restart that effort.

**Comment by chrisn** [comment]
Thank you, and I agree it's sad to drop this, we had some good input. Of course, we're open to restarting work if there's interest.

**Comment by chrisn** [comment]
Would it be helpful to rephrase "tighter support" to something more specific? For example, "achieve a tighter support" in the mission statement could be "to improve support for current and future media related applications and services on the Web".

**Comment by nigelmegitt** [comment]
> Would it be helpful to rephrase "tighter support" to something more specific? For example, "achieve a tighter support" in the mission statement could be "to improve support for current and future media related applications and services on the Web".

Might be worth clarifying the intent of the members here: is it to improve compliance of support with existing specs, to improve the feature set within specs, or a bit of both?

**Comment by chrisn** [comment]
As co-chairs, we are interested to hear wider member feedback. My own view is that it's a bit of both, and can also include proposing where new specs are needed if there are gaps.

**Comment by chrisn** [comment]
The proposed updated mission statement would be:

> "The mission of the Media and Entertainment Interest Group is to provide a forum for media-related technical discussions to track progress of media features on the Web within W3C groups and use of Web technologies by external organizations, and to identify use cases and requirements that existing and/or new specifications need to meet to improve support for current and future media related applications and services on the Web"

**Comment by chrisn** [comment]
@nigelmegitt I think the points you raised may be covered already in the Success Criteria section. Is this OK as is or do you think we need to make a change? We have:

* "Engagement and coordination with other organizations in the media industry to promote adherence to W3C standards"

and 

* "Identification of requirements for improved support of media-centric applications on the Web platform, possibly through the creation of Task Forces"
* "Incubation of technical solutions for the identified requirements by supporting the creation of or discussion in existing Community Groups or Business Groups focusing on particular topics"

**Comment by nigelmegitt** [comment]
@chrisn I agree. I'll suggest an editorial improvement too.

File: charters/charter-2023.html [hunk 1]
```diff
@@ -0,0 +1,876 @@
...
+            application developers, regulators, and users</li>
+          <li>Members of the Interest Group join relevant Working Groups and drive the
+            development of work items</li>
+          <li>Members of the Interest Group start or join related Community/Business Groups and
+            contribute to creating Interest Group Notes</li>
+          <li>Constructive feedback on W3C deliverables posted for review on the
+            <a href="https://lists.w3.org/Archives/Public/public-web-and-tv/">Media and
+            Entertainment IG mailing list</a></li>
+          <li>Engagement and coordination with other organizations in the media
+            industry to promote adherence to W3C standards</li>
```
Comment (nigelmegitt) [inline charters/charter-2023.html:225]:
```suggestion
            industry to promote implementation of W3C standards,
            including Candidate Recommendations and Recommendations</li>
```

File: charters/charter-2023.html [hunk 2]
```diff
@@ -0,0 +1,876 @@
...
+          <p>The technical development of standards is not in scope for the
+          Interest Group. Technical discussions are expected to take place
+          within the appropriate W3C groups if such a group exists, or within a
+          dedicated Community Group or Business Group when incubation is needed.</p>
+        </section>
+      </section>
+
+      <section id='success-criteria'>
+        <h2>Success Criteria</h2>
+        <p>The Interest Group will have succeeded if it can achieve the following:</p>
```
[the remark this answers is not in the document]
Comment (chrisn) [reply charters/charter-2023.html:211]: Thank you, this is a good point. although we may want alter the wording slightly from what you've suggested here.

**Pull request merged**